framework/generated/khronos_generators/khronos_layer_func_table_generator.py

```diff
@@ -30,32 +30,51 @@ class KhronosLayerFuncTableGenerator():
     Generates C++ function table for Khronos Vulkan API calls exported by the layer.
     """
 
-    def write_layer_func_table_contents(self, skip_func_list, align_col):
+    def write_layer_func_table_contents(self, skip_func_list, align_col, table_name_suffix='', skip_var_definition=False):
         api_data = self.get_api_data()
 
+        table_name = '{}_func_table'.format(api_data.api_name.lower())
+        get_table_name = 'Get{}FuncTable'.format(api_data.api_name)
+
+        skip_func_namespace = '{}_entry'.format(api_data.api_name.lower())
+        if(table_name_suffix):
+            table_name += '_{}'.format(table_name_suffix.lower())
+            skip_func_namespace += '_{}'.format(table_name_suffix.lower())
+            get_table_name += '{}'.format(table_name_suffix)
+
+        # Write the static function to get the table.
         write(
-            'const std::unordered_map<std::string, {}> {}_func_table = {{'.
-            format(api_data.void_func_pointer_type, api_data.api_name.lower()),
-            file=self.outFile
-        )
+            'static const std::unordered_map<std::string, {}> {}() {{'.format(
```

Comment:
This is probably the correct place to make the "shouldn't this be inline" comment?  

Not for this PR, but if this is common included, a .cpp location would be yet better.